core/scm.js

```diff
@@ -72,14 +73,24 @@ const SCHEMA_COMMIT = Joi.object().keys({
         .uri()
         .required()
         .label('Link to commit')
-        .example('https://github.com/scredriver-cd/screwdriver/commit/8843d7f92416211de')
+        .example('https://github.com/screwdriver-cd/screwdriver/commit/8843d7f92416211de')
 }).label('SCM Commit');
 
 const SCHEMA_PR = Joi.object().keys({
     url: Joi.string()
         .uri()
         .label('Link to PR')
-        .example('https://git.ouroath.com/MAILSERVICES/gdpr-event-processor/pull/1')
+        .example('https://github.com/screwdriver-cd/screwdriver/pull/1'),
+    title: Joi.string()
+        .max(512)
+        .label('Title of the pull request'),
+    createTime: Joi.date().iso()
```

Comment:
(just wondering) What's the difference between `Joi.date().iso()` vs `Joi.string().isoDate()`? Since we seem to use the second one everywhere else

Reply:
The former is the "base check" while the latter does more to ensure iso compatibility. For external API, i will prefer using the generic type check.